### PR TITLE
feat(dial pad): remember selected country code

### PR DESCRIPTION
**Pull request by paweldomas**
If there's no country code provided by the backend, then restore the previously selected one.

### CODE REVIEW

File: spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js [hunk 1]
```diff
@@ -307,9 +324,11 @@ function mapDispatchToProps(dispatch) {
  */
 function mapStateToProps(state) {
     const { phoneAuthorizeServiceUrl } = getSpotServicesConfig(state);
+    const countryCode = getCountryCode(state);
+    const mostRecentCountryCode = getMostRecentCountryCode(state);
 
     return {
-        countryCode: getCountryCode(state),
+        countryCode: countryCode || mostRecentCountryCode,
```
Comment (virtuacoplenny) [inline spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js]: should mostRecentCountryCode take precedence?

Comment (paweldomas) [reply spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js]: Yeah, at some point I was thinking the same. It's how Emil wanted it though. Also if you think more about it, Spot TV is not a personal device, but shared. So you may not want to call to the country another person was calling to in the meeting before.

Comment (virtuacoplenny) [reply spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js]: Is this prop more of a default country code then? I wonder if renaming it such would be better.

Comment (paweldomas) [reply spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js]: No, it is not default. It is used as default only if nothing's been configured on the backend.

**Pull request merged**